<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="AggregateException.xml" source-language="en-US" target-language="zh-TW">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5a408cd8c65b6a1872bbc7ecc28fd7106d4fe9897.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">a408cd8c65b6a1872bbc7ecc28fd7106d4fe9897</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.AggregateException">
          <source>Represents one or more errors that occur during application execution.</source>
          <target state="translated">代表應用程式執行期間所發生的一或多個錯誤。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source><ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> is used to consolidate multiple failures into a single, throwable exception object.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> 用來將多項失敗合併成單一、 throwable 例外狀況物件。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source>It is used extensively in the <bpt id="p1">[</bpt>Task Parallel Library (TPL)<ept id="p1">](~/docs/standard/parallel-programming/task-parallel-library-tpl.md)</ept> and <bpt id="p2">[</bpt>Parallel LINQ (PLINQ)<ept id="p2">](~/docs/standard/parallel-programming/parallel-linq-plinq.md)</ept>.</source>
          <target state="translated">它廣泛用於<bpt id="p1">[</bpt>工作平行程式庫 (TPL)<ept id="p1">](~/docs/standard/parallel-programming/task-parallel-library-tpl.md)</ept>和<bpt id="p2">[</bpt>平行 LINQ (PLINQ)<ept id="p2">](~/docs/standard/parallel-programming/parallel-linq-plinq.md)</ept>。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source>For additional information, see the <bpt id="p1">[</bpt>Aggregating Exceptions<ept id="p1">](http://go.microsoft.com/fwlink/p/?LinkId=259617)</ept> entry in the .NET Matters blog.</source>
          <target state="translated">如需詳細資訊，請參閱<bpt id="p1">[</bpt>彙總的例外狀況<ept id="p1">](http://go.microsoft.com/fwlink/p/?LinkId=259617)</ept>.NET 複雜化部落格中的項目。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source>For an example, see <bpt id="p1">[</bpt>NIB: How to: Handle Exceptions Thrown by Tasks<ept id="p1">](http://msdn.microsoft.com/library/d6c47ec8-9de9-4880-beb3-ff19ae51565d)</ept> and <bpt id="p2">[</bpt>How to: Handle Exceptions in a PLINQ Query<ept id="p2">](~/docs/standard/parallel-programming/how-to-handle-exceptions-in-a-plinq-query.md)</ept>.</source>
          <target state="translated">如需範例，請參閱<bpt id="p1">[</bpt>NIB： 如何： 處理例外狀況擲回的工作<ept id="p1">](http://msdn.microsoft.com/library/d6c47ec8-9de9-4880-beb3-ff19ae51565d)</ept>和<bpt id="p2">[</bpt>如何： 處理 PLINQ 查詢中的例外狀況<ept id="p2">](~/docs/standard/parallel-programming/how-to-handle-exceptions-in-a-plinq-query.md)</ept>。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source>The following example catches the <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> exception and calls the <ph id="ph2">&lt;xref:System.AggregateException.Handle%2A&gt;</ph> method to handle each exception it contains.</source>
          <target state="translated">下列範例會攔截<ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>例外狀況及呼叫<ph id="ph2">&lt;xref:System.AggregateException.Handle%2A&gt;</ph>方法以處理它所包含的每個例外狀況。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source>Compiling and running the example with the first <ph id="ph1">`task1`</ph> variable should result in an <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> object that contains an <ph id="ph3">&lt;xref:System.UnauthorizedAccessException&gt;</ph> exception.</source>
          <target state="translated">編譯並執行此範例的第一個<ph id="ph1">`task1`</ph>變數應該會導致<ph id="ph2">&lt;xref:System.AggregateException&gt;</ph>物件，其中包含<ph id="ph3">&lt;xref:System.UnauthorizedAccessException&gt;</ph>例外狀況。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source>Commenting out that line, uncommenting the second <ph id="ph1">`task1`</ph> variable, and compiling and running the example produces an <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> object that contains an <ph id="ph3">&lt;xref:System.IndexOutOfRangeException&gt;</ph> exception.</source>
          <target state="translated">標記為註解此行，取消註解第二個<ph id="ph1">`task1`</ph>變數，並編譯和執行範例會產生<ph id="ph2">&lt;xref:System.AggregateException&gt;</ph>物件，其中包含<ph id="ph3">&lt;xref:System.IndexOutOfRangeException&gt;</ph>例外狀況。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.AggregateException">
          <source>All public and protected members of <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> are thread-safe and may be used concurrently from multiple threads.</source>
          <target state="translated">所有的 public 和 protected 成員的<ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph>是安全執行緒，且可能會從多個執行緒同時使用。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="T:System.AggregateException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with a system-supplied message that describes the error.</source>
          <target state="translated">以系統提供的錯誤說明訊息，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "System.AggregateException: One or more errors occurred."</source>
          <target state="translated">這個建構函式初始化<ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph>屬性的新執行個體的系統提供的訊息描述錯誤，例如"System.AggregateException： 發生一個或多個錯誤。 」</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">此訊息會考量目前的系統文化特性。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>.</source>
          <target state="translated">下表顯示 <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> 執行個體的初始屬性值。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor">
          <source>Property</source>
          <target state="translated">屬性</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor">
          <source>Value</source>
          <target state="translated">值</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor">
          <source><ph id="ph1">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">`null`</ph>。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor">
          <source>The localized error message string.</source>
          <target state="translated">當地語系化的錯誤訊息字串。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Collections.Generic.IEnumerable{System.Exception})">
          <source>The exceptions that are the cause of the current exception.</source>
          <target state="translated">導致目前例外狀況的例外。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Collections.Generic.IEnumerable{System.Exception})">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with references to the inner exceptions that are the cause of this exception.</source>
          <target state="translated">使用造成這個例外狀況原因的內部例外狀況參考，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Collections.Generic.IEnumerable{System.Exception})">
          <source>The <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Collections.Generic.IEnumerable{System.Exception})">
          <source>An element of <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 的項目為 Null。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Exception[])">
          <source>The exceptions that are the cause of the current exception.</source>
          <target state="translated">導致目前例外狀況的例外。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Exception[])">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with references to the inner exceptions that are the cause of this exception.</source>
          <target state="translated">使用造成這個例外狀況原因的內部例外狀況參考，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Exception[])">
          <source>The <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Exception[])">
          <source>An element of <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 的項目為 Null。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String)">
          <source>The message that describes the exception.</source>
          <target state="translated">說明例外狀況的訊息。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">這個建構函式的呼叫端必須確保這個字串已經被當地語系化 (為了目前系統的文化特性)。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with a specified message that describes the error.</source>
          <target state="translated">使用指定的錯誤說明訊息，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>.</source>
          <target state="translated">下表顯示 <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> 執行個體的初始屬性值。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String)">
          <source>Property</source>
          <target state="translated">屬性</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String)">
          <source>Value</source>
          <target state="translated">值</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String)">
          <source><ph id="ph1">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">`null`</ph>。</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String)">
          <source>The error message string specified in <ph id="ph1">`message`</ph>.</source>
          <target state="translated"><ph id="ph1">`message`</ph> 中指定的錯誤訊息字串。</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">存放序列物件資料的物件。</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">關於來源或目的端的內容資訊。</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">使用序列化資料，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">進行還原序列化 (Deserialization) 期間會呼叫這個建構函式，以便重新構成經由資料流傳送的例外狀況物件。</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">如需詳細資訊，請參閱<bpt id="p1">[</bpt>XML 和 SOAP 序列化<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>。</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;paramref name="info" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="info" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The exception could not be deserialized correctly.</source>
          <target state="translated">無法將例外狀況正確還原序列化。</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Collections.Generic.IEnumerable{System.Exception})">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">解釋例外狀況原因的錯誤訊息。</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Collections.Generic.IEnumerable{System.Exception})">
          <source>The exceptions that are the cause of the current exception.</source>
          <target state="translated">導致目前例外狀況的例外。</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Collections.Generic.IEnumerable{System.Exception})">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with a specified error message and references to the inner exceptions that are the cause of this exception.</source>
          <target state="translated">使用指定的錯誤訊息和造成這個例外狀況原因的內部例外狀況參考，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Collections.Generic.IEnumerable{System.Exception})">
          <source>The <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Collections.Generic.IEnumerable{System.Exception})">
          <source>An element of <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 的項目為 Null。</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>The message that describes the exception.</source>
          <target state="translated">說明例外狀況的訊息。</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">這個建構函式的呼叫端必須確保這個字串已經被當地語系化 (為了目前系統的文化特性)。</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">做為目前例外狀況發生原因的例外狀況。</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">如果 <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> 參數不是 <ph id="ph1">&lt;see langword="null" /&gt;</ph>，則目前的例外狀況會在處理內部例外的 <ph id="ph2">&lt;see langword="catch" /&gt;</ph> 區塊中引發。</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">使用指定的錯誤訊息以及造成此例外狀況的內部例外狀況的參考，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">被擲回以做為前一個例外狀況直接結果的例外狀況，應該在 <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 屬性中包含對前一個例外狀況的參考。</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 屬性會傳回與傳入建構函式中相同的值；如果 <ph id="ph2">`null`</ph> 屬性沒有提供內部例外值給建構函式，則傳回 <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>.</source>
          <target state="translated">下表顯示 <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> 執行個體的初始屬性值。</target>       </trans-unit>
        <trans-unit id="155" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">屬性</target>       </trans-unit>
        <trans-unit id="156" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">值</target>       </trans-unit>
        <trans-unit id="157" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source><ph id="ph1">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">`null`</ph>。</target>       </trans-unit>
        <trans-unit id="158" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>The error message string specified in <ph id="ph1">`message`</ph>.</source>
          <target state="translated"><ph id="ph1">`message`</ph> 中指定的錯誤訊息字串。</target>       </trans-unit>
        <trans-unit id="159" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;paramref name="innerException" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerException" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="160" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception[])">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">解釋例外狀況原因的錯誤訊息。</target>       </trans-unit>
        <trans-unit id="161" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception[])">
          <source>The exceptions that are the cause of the current exception.</source>
          <target state="translated">導致目前例外狀況的例外。</target>       </trans-unit>
        <trans-unit id="162" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception[])">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with a specified error message and references to the inner exceptions that are the cause of this exception.</source>
          <target state="translated">使用指定的錯誤訊息和造成這個例外狀況原因的內部例外狀況參考，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="163" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception[])">
          <source>The <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="164" translate="yes" xml:space="preserve" uid="M:System.AggregateException.#ctor(System.String,System.Exception[])">
          <source>An element of <ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="innerExceptions" /&gt;</ph> 的項目為 Null。</target>       </trans-unit>
        <trans-unit id="165" translate="yes" xml:space="preserve" uid="M:System.AggregateException.Flatten">
          <source>Flattens an <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> instances into a single, new instance.</source>
          <target state="translated">將 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 執行個體簡維成單一新執行個體。</target>       </trans-unit>
        <trans-unit id="166" translate="yes" xml:space="preserve" uid="M:System.AggregateException.Flatten">
          <source>A new, flattened <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph>.</source>
          <target state="translated">簡維的新 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="167" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Flatten">
          <source>If a task has an attached child task that throws an exception, that exception is wrapped in an <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> exception before it is propagated to the parent task, which wraps that exception in its own <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> exception before it propagates it back to the calling thread.</source>
          <target state="translated">如果工作有附加的子工作擲回例外狀況，該例外狀況會包裝在<ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>之前再傳播到父工作中，例外狀況包裝在它自己的例外狀況<ph id="ph2">&lt;xref:System.AggregateException&gt;</ph>例外狀況之前它會傳播回呼叫執行緒。</target>       </trans-unit>
        <trans-unit id="168" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Flatten">
          <source>In such cases, the <ph id="ph1">&lt;xref:System.AggregateException.InnerExceptions%2A&gt;</ph> property of the <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> exception that is caught by the <ph id="ph3">&lt;xref:System.Threading.Tasks.Task.Wait%2A?displayProperty=nameWithType&gt;</ph>, <ph id="ph4">&lt;xref:System.Threading.Tasks.Task.Wait*&gt;</ph>, <ph id="ph5">&lt;xref:System.Threading.Tasks.Task.WaitAny%2A?displayProperty=nameWithType&gt;</ph> or <ph id="ph6">&lt;xref:System.Threading.Tasks.Task.WaitAll%2A?displayProperty=nameWithType&gt;</ph> method contains one or more <ph id="ph7">&lt;xref:System.AggregateException&gt;</ph> instances, not the original exceptions that caused the fault.</source>
          <target state="translated">在這種情況下，<ph id="ph1">&lt;xref:System.AggregateException.InnerExceptions%2A&gt;</ph>屬性<ph id="ph2">&lt;xref:System.AggregateException&gt;</ph>被攔截的例外狀況<ph id="ph3">&lt;xref:System.Threading.Tasks.Task.Wait%2A?displayProperty=nameWithType&gt;</ph>， <ph id="ph4">&lt;xref:System.Threading.Tasks.Task.Wait*&gt;</ph>，<ph id="ph5">&lt;xref:System.Threading.Tasks.Task.WaitAny%2A?displayProperty=nameWithType&gt;</ph>或<ph id="ph6">&lt;xref:System.Threading.Tasks.Task.WaitAll%2A?displayProperty=nameWithType&gt;</ph>方法包含一或多個<ph id="ph7">&lt;xref:System.AggregateException&gt;</ph>執行個體，不原始的例外狀況，導致這個錯誤。</target>       </trans-unit>
        <trans-unit id="169" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Flatten">
          <source>To avoid having to iterate over nested <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> exceptions, you can use the <ph id="ph2">&lt;xref:System.AggregateException.Flatten%2A&gt;</ph> method to remove all the nested <ph id="ph3">&lt;xref:System.AggregateException&gt;</ph> exceptions, so that the <ph id="ph4">&lt;xref:System.AggregateException.InnerExceptions%2A&gt;</ph> property of the returned <ph id="ph5">&lt;xref:System.AggregateException&gt;</ph> object contains the original exceptions.</source>
          <target state="translated">若要避免必須逐一查看巢狀<ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>例外狀況，您可以使用<ph id="ph2">&lt;xref:System.AggregateException.Flatten%2A&gt;</ph>方法移除所有的巢狀<ph id="ph3">&lt;xref:System.AggregateException&gt;</ph>例外狀況，以便<ph id="ph4">&lt;xref:System.AggregateException.InnerExceptions%2A&gt;</ph>屬性傳回之<ph id="ph5">&lt;xref:System.AggregateException&gt;</ph>物件包含原始例外狀況。</target>       </trans-unit>
        <trans-unit id="170" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Flatten">
          <source>This method recursively flattens all instances of  <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> exceptions that are inner exceptions of the current <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> instance .</source>
          <target state="translated">此方法以遞迴方式扁平化的所有執行個體<ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>是目前的內部例外狀況的例外狀況<ph id="ph2">&lt;xref:System.AggregateException&gt;</ph>執行個體。</target>       </trans-unit>
        <trans-unit id="171" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Flatten">
          <source>The inner exceptions returned in the new <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> are the union of all the inner exceptions from exception tree rooted at the current <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> instance.</source>
          <target state="translated">在新的內部例外狀況傳回<ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>都是從例外狀況的樹狀目錄內部的例外狀況的聯集為根目錄目前<ph id="ph2">&lt;xref:System.AggregateException&gt;</ph>執行個體。</target>       </trans-unit>
        <trans-unit id="172" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Flatten">
          <source>In the following example, nested <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> instances are flattened and handled in just one loop.</source>
          <target state="translated">下例中，巢狀 <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> 執行個體被扁平化，並只在一個迴圈中處理。</target>       </trans-unit>
        <trans-unit id="173" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Flatten">
          <source>You can also use the <ph id="ph1">&lt;xref:System.AggregateException.Flatten%2A?displayProperty=nameWithType&gt;</ph> method to rethrow the inner exceptions from multiple <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> instances thrown by multiple tasks in a single <ph id="ph3">&lt;xref:System.AggregateException&gt;</ph> instance, as the following example shows.</source>
          <target state="translated">您也可以使用 <ph id="ph1">&lt;xref:System.AggregateException.Flatten%2A?displayProperty=nameWithType&gt;</ph> 方法，從被單一 <ph id="ph3">&lt;xref:System.AggregateException&gt;</ph> 執行個體之多個工作擲回的多個 <ph id="ph2">&lt;xref:System.AggregateException&gt;</ph> 執行個體中，重新擲回內部的例外狀況，如下例所示。</target>       </trans-unit>
        <trans-unit id="174" translate="yes" xml:space="preserve" uid="M:System.AggregateException.GetBaseException">
          <source>Returns the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> that is the root cause of this exception.</source>
          <target state="translated">傳回這個例外狀況之根本原因的 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="175" translate="yes" xml:space="preserve" uid="M:System.AggregateException.GetBaseException">
          <source>Returns the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> that is the root cause of this exception.</source>
          <target state="translated">傳回這個例外狀況之根本原因的 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="176" translate="yes" xml:space="preserve" uid="M:System.AggregateException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">存放序列物件資料的物件。</target>       </trans-unit>
        <trans-unit id="177" translate="yes" xml:space="preserve" uid="M:System.AggregateException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">關於來源或目的端的內容資訊。</target>       </trans-unit>
        <trans-unit id="178" translate="yes" xml:space="preserve" uid="M:System.AggregateException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">使用序列化資料，初始化 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="179" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">進行還原序列化 (Deserialization) 期間會呼叫這個建構函式，以便重新構成經由資料流傳送的例外狀況物件。</target>       </trans-unit>
        <trans-unit id="180" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">如需詳細資訊，請參閱<bpt id="p1">[</bpt>XML 和 SOAP 序列化<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>。</target>       </trans-unit>
        <trans-unit id="181" translate="yes" xml:space="preserve" uid="M:System.AggregateException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;paramref name="info" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="info" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="182" translate="yes" xml:space="preserve" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>The predicate to execute for each exception.</source>
          <target state="translated">要針對每個例外狀況執行的述詞。</target>       </trans-unit>
        <trans-unit id="183" translate="yes" xml:space="preserve" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>The predicate accepts as an argument the <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> to be processed and returns a Boolean to indicate whether the exception was handled.</source>
          <target state="translated">述詞會接受要處理的 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> 做為引數，並傳回表示是否已處理例外狀況的布林值。</target>       </trans-unit>
        <trans-unit id="184" translate="yes" xml:space="preserve" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>Invokes a handler on each <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> contained by this <ph id="ph2">&lt;see cref="T:System.AggregateException" /&gt;</ph>.</source>
          <target state="translated">對這個 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> 包含的每個 <ph id="ph2">&lt;see cref="T:System.AggregateException" /&gt;</ph> 叫用處理常式。</target>       </trans-unit>
        <trans-unit id="185" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>Each invocation of the <ph id="ph1">`predicate`</ph> returns true or false to indicate whether the <ph id="ph2">&lt;xref:System.Exception&gt;</ph> was handled.</source>
          <target state="translated">每次叫用<ph id="ph1">`predicate`</ph>，則傳回 true 或 false，指出是否<ph id="ph2">&lt;xref:System.Exception&gt;</ph>已處理。</target>       </trans-unit>
        <trans-unit id="186" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>After all invocations, if any exceptions went unhandled, all unhandled exceptions will be put into a new <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> which will be thrown.</source>
          <target state="translated">所有的引動過程之後, 如果任何例外狀況發生未處理，所有未處理的例外狀況將會進入新<ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>這將會擲回。</target>       </trans-unit>
        <trans-unit id="187" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>Otherwise, the <ph id="ph1">&lt;xref:System.AggregateException.Handle%2A&gt;</ph> method simply returns.</source>
          <target state="translated">否則，<ph id="ph1">&lt;xref:System.AggregateException.Handle%2A&gt;</ph>方法只會傳回。</target>       </trans-unit>
        <trans-unit id="188" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>If any invocations of the <ph id="ph1">`predicate`</ph> throws an exception, it will halt the processing of any more exceptions and immediately propagate the thrown exception as-is.</source>
          <target state="translated">如果任何引動過程的<ph id="ph1">`predicate`</ph>擲回例外狀況，則會停止任何多個例外狀況處理，並立即傳播做為擲回的例外狀況-是。</target>       </trans-unit>
        <trans-unit id="189" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>Ordinarily, an exception handler that catches an <ph id="ph1">&lt;xref:System.AggregateException&gt;</ph> exception uses a <ph id="ph2">`foreach`</ph> loop (in C#) or <ph id="ph3">`For Each`</ph> loop (in Visual Basic) to handle each exception in its <ph id="ph4">&lt;xref:System.AggregateException.InnerExceptions%2A&gt;</ph> collection.</source>
          <target state="translated">一般情況下，會攔截的例外狀況處理常式<ph id="ph1">&lt;xref:System.AggregateException&gt;</ph>例外狀況會使用<ph id="ph2">`foreach`</ph>迴圈 （在 C# 中) 或<ph id="ph3">`For Each`</ph>迴圈 （在 Visual Basic) 來處理每個例外狀況，在其<ph id="ph4">&lt;xref:System.AggregateException.InnerExceptions%2A&gt;</ph>集合。</target>       </trans-unit>
        <trans-unit id="190" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>Instead, the following example uses the <ph id="ph1">&lt;xref:System.AggregateException.Handle%2A&gt;</ph> method to handle each exception, and only re-throws exceptions that are not <ph id="ph2">`CustomException`</ph> instances.</source>
          <target state="translated">相反地，下列範例使用<ph id="ph1">&lt;xref:System.AggregateException.Handle%2A&gt;</ph>方法以處理每個例外狀況，而不只會重新擲回的例外<ph id="ph2">`CustomException`</ph>執行個體。</target>       </trans-unit>
        <trans-unit id="191" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>The following is a more complete example that uses the <ph id="ph1">&lt;xref:System.AggregateException.Handle%2A&gt;</ph> method to provide special handling for an <ph id="ph2">&lt;xref:System.UnauthorizedAccessException&gt;</ph> when enumerating files.</source>
          <target state="translated">以下是更完整的範例，其使用<ph id="ph1">&lt;xref:System.AggregateException.Handle%2A&gt;</ph>方法以提供的特殊處理<ph id="ph2">&lt;xref:System.UnauthorizedAccessException&gt;</ph>列舉檔案時。</target>       </trans-unit>
        <trans-unit id="192" translate="yes" xml:space="preserve" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>The <ph id="ph1">&lt;paramref name="predicate" /&gt;</ph> argument is null.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="predicate" /&gt;</ph> 引數為 null。</target>       </trans-unit>
        <trans-unit id="193" translate="yes" xml:space="preserve" uid="M:System.AggregateException.Handle(System.Func{System.Exception,System.Boolean})">
          <source>An exception contained by this <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> was not handled.</source>
          <target state="translated">未處理此 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 所包含的例外狀況。</target>       </trans-unit>
        <trans-unit id="194" translate="yes" xml:space="preserve" uid="P:System.AggregateException.InnerExceptions">
          <source>Gets a read-only collection of the <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> instances that caused the current exception.</source>
          <target state="translated">取得造成目前例外狀況之 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> 執行個體的唯讀集合。</target>       </trans-unit>
        <trans-unit id="195" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.AggregateException.InnerExceptions">
          <source>Returns a read-only collection of the <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> instances that caused the current exception.</source>
          <target state="translated">傳回造成目前例外狀況之 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> 執行個體的唯讀集合。</target>       </trans-unit>
        <trans-unit id="196" translate="yes" xml:space="preserve" uid="M:System.AggregateException.ToString">
          <source>Creates and returns a string representation of the current <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph>.</source>
          <target state="translated">建立並傳回目前 <ph id="ph1">&lt;see cref="T:System.AggregateException" /&gt;</ph> 的字串表示。</target>       </trans-unit>
        <trans-unit id="197" translate="yes" xml:space="preserve" uid="M:System.AggregateException.ToString">
          <source>A string representation of the current exception.</source>
          <target state="translated">目前例外狀況的字串表示。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>